datasource/opennms/src/main/java/org/opennms/oce/datasource/opennms/OpennmsDatasource.java

```diff
@@ -326,7 +329,23 @@ public void unregisterHandler(SituationHandler handler) {
         } catch (InterruptedException e) {
             throw new RuntimeException(e);
         }
-        return InventoryTableProcessor.toInventory(inventory);
+
+        Set<InventoryIdentifier> uniqueIds = new HashSet<>();
```

Comment:
I just remembered... we already have a `ResourceKey` class that can be used for this purpose